parm/evs_config/aqm/config.evs.aqm.prod

```diff
@@ -37,9 +37,9 @@ export end_date=$VDATE
 #metplus_verbosity:         DEBUG, INFO, WARN, ERROR (most verbose is DEBUG)
 #met_verbosity:             MET verbosity for logging, 1-5 (most verbose is 5)
 #log_met_output_to_metplus: log MET output to METplus log, yes or no
-export metplus_verbosity="DEBUG"
-export met_verbosity="2"
-export log_met_output_to_metplus="yes"
+# export metplus_verbosity="DEBUG"
+# export met_verbosity="2"
+# export log_met_output_to_metplus="yes"
 ## DATA DIRECTIVE SETTINGS
```

Comment:
@PerryShafran-NOAA , @Ho-ChunHuang-NOAA 
Should lines 40-42 be wholesale removed, or is leaving them in, but commented out, OK?